src/screens/search_results/SearchResultsHeader.tsx

```diff
@@ -92,11 +103,46 @@ const SearchResultsHeader: React.FC<SearchResultsHeaderProps> = ({
                 Search results for: <span className='underline'>{query}</span>
             </Typ>
             <div>
+                <ToggleButtonGroup value={filterState.showType} exclusive sx={{ marginRight: 2 }}>
+                    <ToggleButton
+                        value='tv'
+                        aria-label='filter by tv shows'
+                        onClick={() => {
+                            setFilterState({ showType: 'tv' });
+                        }}
+                        disabled={disableControls}
+                    >
+                        <Tv />
+                    </ToggleButton>
+                    <ToggleButton
+                        value='movie'
+                        aria-label='filter by movies'
+                        onClick={() => {
+                            setFilterState({ showType: 'movie' });
+                        }}
+                        disabled={disableControls}
+                    >
+                        <Movie />
+                    </ToggleButton>
+                    <ToggleButton
+                        value='none'
+                        aria-label='Remove filter'
+                        onClick={() => {
+                            setFilterState({ showType: 'none' });
+                        }}
+                        disabled={disableControls}
+                    >
+                        <FilterAltOff />
+                    </ToggleButton>
+                </ToggleButtonGroup>
+
                 <ToggleButtonGroup value={sortState} exclusive sx={{ marginRight: 2 }}>
                     <ToggleButton
                         value='alpha'
                         aria-label='sort results alphabetically'
-                        onClick={handleSortAlpha}
+                        onClick={() => {
+                            setSortState('alpha');
+                        }}
```

Comment:
```suggestion
                        onClick={() => setSortState('alpha')}
```
Any chance es-lint will let some of these slide on a single line? lol